swap/src/bin/asb.rs

```diff
@@ -140,29 +170,13 @@ async fn main() -> Result<()> {
                 }
             }
 
-            let bitcoin_wallet = init_bitcoin_wallet(&config, &seed, env_config).await?;
+            let bitcoin_wallet =
+                init_bitcoin_wallet(&config, &seed, env_config, proxy_string).await?;
```

Comment:
if the env_config has the tor port then this additional argument is not necessary